angular2/src/angular2TextMask.ts

```diff
@@ -45,11 +45,11 @@ export class MaskedInputDirective implements ControlValueAccessor {
       this.inputElement = this.element.nativeElement.getElementsByTagName('INPUT')[0]
     }
 
-    if (this.inputElement) {
-      this.textMaskInputElement = createTextMaskInputElement(
-          Object.assign({inputElement: this.inputElement}, this.textMaskConfig)
-      )
-    }
+    this.textMaskInputElement = createTextMaskInputElement({})
```

Comment:
@PerfectPixel how can we change the [typescript definition for text-mask-core](https://github.com/text-mask/text-mask/blob/master/core/src/createTextMaskInputElement.d.ts#L1) so that the config is optional

```js
export default function createTextMaskInputElement(a: any): any
```

so the `a` argument in `createTextMaskInputElement(a: any)` is optional.